vuepress/docs/next/tutorials/devops/external-database/README.md

```diff
@@ -20,40 +20,100 @@ backup workflow that will scale with your application.
 
 ## Tutorial
 
-1.  In the helm quickstart define a db for your deployment that includes
-    the admin credentials for your database
-
-<!-- -->
-
-    db:
-      vendor: postgresql | mysql | oracle
-      host: <yourhost>
-      port: <yourport>
-      adminUser: <youradmin>
-      adminPassword: <youradminpassword>
-      name: <yourdbname>
-
-1.  Run the helm quickstart or create a deployment for your Entando
-    environment manually
-
-    1.  Note: The quickstart creates a default configuration for the
-        deployment. This isn’t to suggest that it is the only
-        configuration or that it matches with every use case. It is
-        recommended to think about your desired deployment Architecture
-        and to configure for that environment using the CRDs as building
-        blocks. This also isn’t to suggest that if it isn’t directly
-        supported by the quickstart template that you can’t change it.
-        The deployment is editable. Treat the helm template as a
-        bootstrapping environment and edit your deployments to match
-        your needs and requirements.
-
-2.  Open the yaml file generated from the deployment and review the
-    settings for the deployment
-
-    1.  The Entando Operator will create a secret for the database
-        automatically and use the DB initializers to create the DB for
-        the Entando app and any services that use it
-
-3.  Deploy your app
-
-
+1. Create and install a secret or secrets for your database credentials
+2. Edit the deployment template generated from running the helm command or the helm template you used to deploy your Entando application
+3. Find the entry for the `EntandoCompositeApp`
+4. Set the value for `dbms` to `none`
+5. Under `environmentVariables` you will need to provide database connection information for two Entando Server database and one connection for the Entando Component Repository. Those connects  are:
```

Comment:
1) databases, plural. 2) "Those connects" is a little awkward.

Reply:
Done.